New Feature: Allow user to create entry.

jojobii-arks opened the pull request:
## resolves #1 
![Animation](https://user-images.githubusercontent.com/78003700/161649161-54298b27-0af4-4a9f-83d2-44d04b61bd6f.gif)

jojobii-arks committed:
initial form created

jojobii-arks committed:
Completed styling for New Entry form

jojobii-arks committed:
added event listener to update preview image
added submit handler/listener to modify data object with incrementing entryId
added data saving to localStorage via JSON

jojobii-arks committed:
changed the way submitHandler() resets the image preview

codingcodymiller commented on css/styles.css:
```diff
@@ -0,0 +1,90 @@
+  height: 250px;
+}
+
+@media screen and (min-width: 768px) {
+  img.entry-thumbnail {
+    height: 100%;
+  }
+}
```
Make sure you always have media queries as the last thing in a CSS file. There should be no general CSS rules below a media query ever.

codingcodymiller commented on index.html:
```diff
@@ -15,10 +18,60 @@
-  <header>
+
+  <header class="txt-color-white">
+    <div class="row container logo">
```
No one element should be both a `container` and a `row`. You should make the `row` a child of this element.

codingcodymiller commented on js/data.js:
```diff
@@ -1,8 +1,22 @@
+  localStorage.setItem('data', JSON.stringify(data));
+}
+
+var storedData = localStorage.getItem('data');
```
You should move this inside the function definition.

jojobii-arks committed:
resolved requests:
- moved media queries below other css rulesets
- index.html: made '.row' a child of '.container.logo'
- js/data.js: moved storedData variable declaration into function
